app/helpers/container_project_helper/textual_summary.rb

```diff
@@ -16,6 +16,61 @@ def textual_group_smart_management
     items.collect { |m| send("textual_#{m}") }.flatten.compact
   end
 
+  def textual_quota
+    {
+      :labels => [_("Quota Name"), _("Resource"), _("Desired"), _("Enforced"), _("Observed")],
+      :values => aggregate_quota_items_by_resource
+    }
+  end
+
+  def aggregate_quota_items_by_resource
+    rows = []
+    @record.container_quotas.each do |quota|
+      quota.container_quota_items.each do |item|
```

Comment:
@moolitayer I think you can add to `ContainerProject` something like:

    has_many :container_quota_items, :through => :container_quotas

So that you spare a nested loop here.